gdsc-web-backend/Controllers/EventController.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Collections.Generic;
+using gdsc_web_backend.Models;
+using gdsc_web_backend.Models.Enums;
+using Microsoft.AspNetCore.Mvc;
+
+namespace gdsc_web_backend.Controllers
+{
+    // This marks this controller as a public one that can be called from the internet
+    [ApiController]  
+    // This sets the URL that we can enter to call the controller's methods
+    // ex: https://localhost:5000/api/Example
+    [Route("api/[controller]")]  
+    public class EventController : ControllerBase
+    {
+        /// <summary>
```

Comment:
Please remove these comments, since they are the same like in the ExampleModel.
Use comments only if they are relevant to the current file

Reply:
Also, rename EventController to EventsController because a controller manages more events

Reply:
Solved that, but I thought I would stick the commit to this pull request by clicking a merge button here on the site, but I think I accidentally merged it directly?

Reply:
Yes, you've merged pull request, but that's okay, there are no breaking changes :)))

Reply:
You can remove revert-3-event-model branch and commit